nbformat/tests/test_sign.py

```diff
@@ -28,7 +28,16 @@ def setUp(self):
     
     def tearDown(self):
         shutil.rmtree(self.data_dir)
-    
+   
+    def test_invalid_db_file(self):
+        invalid_notary = sign.NotebookNotary(
+            db_file='invalid_db_file.db',
+            secret=b'secret',
+            data_dir='.',
```

Comment:
For some reason...I can't get this code to throw an exception when I run the tests.  Even in removing the code I modified in `sign.py`, it wouldn't throw an exception.  Is there something more sensible I should do to get throw an exception?